### PR TITLE
Update top-level README file

### PR DESCRIPTION
**Pull request by namurphy**
This PR includes several updates to PlasmaPy's principal README file, including updates to our badges.  I did my best guess about the badges that we dropped, though I wasn't completely sure we have stopped using Codecov.  Any that I shouldn't have dropped we can put back in there.  If we add any new GitHub Actions workflows, we should add a badge for them here too.  I still want to add a badge for the YouTube channel, and then put a bit more information about our weekly events.

### CODE REVIEW

**Comment by StanczakDominik** [comment]
Codecov is still used!

File: README.md [hunk 1]
```diff
@@ -2,32 +2,34 @@
...
+
+[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.1436011.svg)](https://doi.org/10.5281/zenodo.1436011)
 [![astropy](http://img.shields.io/badge/powered%20by-AstroPy-orange.svg?style=flat)](http://www.astropy.org/)
+[![pre-commit](https://img.shields.io/badge/pre--commit-enabled-brightgreen?logo=pre-commit&logoColor=white)](https://github.com/pre-commit/pre-commit)
 [![Open Source Helpers](https://www.codetriage.com/plasmapy/plasmapy/badges/users.svg)](https://www.codetriage.com/plasmapy/plasmapy)
 
 PlasmaPy is an open source community developed Python 3.6+ package for
-plasma physics in the early stages of development.  PlasmaPy intends to
+plasma physics that is currently under development.  PlasmaPy intends to
 be for plasmas what [Astropy](https://github.com/astropy/astropy) is for
```
Comment (Tiger-Du) [inline README.md:26]: Should "plasmas" be changed to "plasma physics" for consistency with "astronomy"?

File: README.md [hunk 2]
```diff
@@ -44,6 +46,18 @@ on our
...
+We have weekly community meetings in the
+[PlasmaPy room on Jitsi](https://meet.jit.si/plasmapy).
+The schedule of our community meetings is on our [calendar](https://calendar.google.com/calendar?cid=bzVsb3ZkcW0zaWxsam00ZTlrMDd2cmw5bWdAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ), and you may access the [minutes and
+agendas](https://drive.google.com/drive/folders/0ByPG8nie6fTPV1FQUEkzMTgtRTg?usp=sharing).
+Any last minute changes will be discussed on
+[Matrix](https://app.element.io/app/#/room/#plasmapy:openastronomy.org).
+As of January 2021, our meetings are on Tuesdays at
+[19:00 UTC](http://time.unitarium.com/utc/6pm).
+Come discuss plasma software with us!
+
```
Comment (Tiger-Du) [inline README.md:60]: Should the weekly office hours be mentioned here as well?

File: README.md [hunk 3]
```diff
@@ -2,32 +2,34 @@
...
+[![GitHub Actions — Style linters](https://github.com/PlasmaPy/PlasmaPy/workflows/Style%20linters/badge.svg)](https://github.com/PlasmaPy/PlasmaPy/actions?query=workflow%3AStyle-linters+branch%3Amaster)
+[![Read the Docs Status](https://readthedocs.org/projects/plasmapy/badge/?version=latest)](http://plasmapy.readthedocs.io/en/latest/?badge=latest)
+
+
+[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.1436011.svg)](https://doi.org/10.5281/zenodo.1436011)
 [![astropy](http://img.shields.io/badge/powered%20by-AstroPy-orange.svg?style=flat)](http://www.astropy.org/)
+[![pre-commit](https://img.shields.io/badge/pre--commit-enabled-brightgreen?logo=pre-commit&logoColor=white)](https://github.com/pre-commit/pre-commit)
 [![Open Source Helpers](https://www.codetriage.com/plasmapy/plasmapy/badges/users.svg)](https://www.codetriage.com/plasmapy/plasmapy)
 
 PlasmaPy is an open source community developed Python 3.6+ package for
```
Comment (Tiger-Du) [inline README.md:24]: Just a punctuation suggestion that might improve readability: "...open-source, community-developed Python 3.6+ package...".

**Review by Tiger-Du — COMMENTED** [review]
In addition to the suggestions below, it might be nice to update "Feedback and communication" to include "Community" somehow. Maybe "Community and feedback", "Community and communication", or "Community". Just my thoughts.

File: README.md [hunk 4]
```diff
@@ -2,40 +2,61 @@
...
+[![PyPI version](https://img.shields.io/pypi/pyversions/plasmapy?style=flat&logo=python)](https://img.shields.io/pypi/pyversions/plasmapy?style=plastic)
 [![License](https://img.shields.io/badge/License-BSD%203--Clause-blue.svg)](./LICENSE.md)
-[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.1436011.svg)](https://doi.org/10.5281/zenodo.1436011)
 
-[![Build Status](https://travis-ci.org/PlasmaPy/PlasmaPy.svg?branch=master)](https://travis-ci.org/PlasmaPy/PlasmaPy)
-[![Build Status](https://dev.azure.com/plasmapy/PlasmaPy/_apis/build/status/PlasmaPy.PlasmaPy?branchName=master)](https://dev.azure.com/plasmapy/PlasmaPy/_build/latest?definitionId=2&branchName=master)
-[![Build status](https://ci.appveyor.com/api/projects/status/hbduy62sqrvy8rn7?svg=true)](https://ci.appveyor.com/project/namurphy/plasmapy)
+[![Matrix](https://img.shields.io/badge/Matrix-join%20chat-blueviolet?style=flat&logo=matrix)](https://app.element.io/#/room/#plasmapy:openastronomy.org)
+[![Matrix](https://img.shields.io/badge/gitter-join%20chat-emerald?style=flat&logo=gitter)](https://gitter.im/PlasmaPy/Lobby)
```
Comment (StanczakDominik) [inline README.md:11]:
```suggestion
```
While the two rooms were already bridged before, [Gitter is now actually a front-end for Matrix](https://matrix.org/blog/2020/09/30/welcoming-gitter-to-matrix), so it probably makes sense to have a single chat badge here.

File: README.md [hunk 5]
```diff
@@ -2,40 +2,61 @@
...
-astronomy — a collection of functionality commonly used and shared
-between plasma physicists and researchers globally, running within and
-leveraging the open source scientific Python ecosystem.  The goals of
-this project are more thoroughly described in our [vision
-statement](http://docs.plasmapy.org/en/stable/about/vision_statement.html)
-and [this recent reference](https://doi.org/10.5281/zenodo.1238132).
-We are in the process of writing [online
+[PlasmaPy](https://www.plasmapy.org/) is an open source community
+developed Python package for plasma science that is currently under
+development.PlasmaPy intends to be for plasma science what
```
Comment (StanczakDominik) [inline README.md:28]:
```suggestion
[PlasmaPy](https://www.plasmapy.org/) is an open source, community-developed
Python package for plasma science that is currently under
development. PlasmaPy intends to be for plasma science what
```
Incorporated suggestion from @Tiger-Du. Also, the second part of that sentence always sounded weird to me - do you expect that we'll eventually stop development?

File: README.md [hunk 6]
```diff
@@ -2,40 +2,61 @@
...
 New contributors are very welcome!
 
-# Feedback and communication
+# Installation
+
+If you have [installed Python](https://wiki.python.org/moin/BeginnersGuide/Download),
+you can install PlasmaPy from [pip](https://pypi.org/project/pip/)
+(sometimes `pip3`) via
+```Shell
+pip install plasmapy
```
Comment (StanczakDominik) [inline README.md:47]: The most universal form that should work everywhere would be
```suggestion
python -m pip install plasmapy
```

File: README.md [hunk 7]
```diff
@@ -2,40 +2,61 @@
...
-and have a [Code of Conduct](http://docs.plasmapy.org/en/stable/CODE_OF_CONDUCT.html).
+and have a [code of conduct](http://docs.plasmapy.org/en/stable/CODE_OF_CONDUCT.html).
 New contributors are very welcome!
 
-# Feedback and communication
+# Installation
+
+If you have [installed Python](https://wiki.python.org/moin/BeginnersGuide/Download),
+you can install PlasmaPy from [pip](https://pypi.org/project/pip/)
+(sometimes `pip3`) via
```
Comment (StanczakDominik) [inline README.md:45]:
```suggestion
via
```

parenthesis obsoleted by note below

**Review by StanczakDominik — APPROVED** [review]
LGTM! A few directly applicable suggestions below that you could apply and merge right away :)

**Pull request merged**